src/signals/IncidentMap/components/IncidentMap/usePaginatedIncidents.ts

```diff
@@ -23,65 +23,77 @@ const usePaginatedIncidents = () => {
   })
 
   const getIncidents = useCallback(async (bbox: Bbox) => {
-    if (bbox) {
-      paginatedIncidents.current.searchParams = getSearchParams(bbox)
-      paginatedIncidents.current.controller?.abort()
-      paginatedIncidents.current.controller = new AbortController()
+    paginatedIncidents.current.searchParams = getSearchParams(bbox)
+    paginatedIncidents.current.controller?.abort()
+    paginatedIncidents.current.controller = new AbortController()
 
-      const { signal } = paginatedIncidents.current.controller
+    const { signal } = paginatedIncidents.current.controller
 
-      let headResponse
-      try {
-        headResponse = await fetch(
+    let headResponse
+    try {
+      headResponse = await fetch(
+        `${
+          configuration.GEOGRAPHY_PUBLIC_ENDPOINT
+        }?${paginatedIncidents.current.searchParams.toString()}`,
+        {
+          method: 'HEAD',
+          signal,
+        }
+      )
+      if (!headResponse.ok) {
+        throw new Error('HTTP status code: ' + headResponse.status)
+      }
+    } catch (error: FetchError | any) {
+      /* istanbul ignore next */
+      if (error.name !== 'AbortError') {
+        setError(error)
+      }
+      return
+    }
+
+    const xTotalCount = Number(headResponse.headers.get('X-Total-Count'))
+    // Would be nice to get this info through the response header as well
+    const paginationLength = 4000
+
+    const promises = Array.from(
+      {
+        length: Math.ceil(xTotalCount / paginationLength),
+      },
+      (_, index) => {
+        const qeopageQueryParam = `&geopage=${index + 1}`
+        return fetch(
           `${
             configuration.GEOGRAPHY_PUBLIC_ENDPOINT
-          }?${paginatedIncidents.current.searchParams.toString()}`,
+          }?${paginatedIncidents.current.searchParams.toString()}${qeopageQueryParam}`,
           {
-            method: 'HEAD',
+            method: 'GET',
```

Comment:
Why did this become a GET request instead of HEAD?

Reply:
Because it needs to return json, not just head info.